libs/api-model-bear/src/meta/GdcMetadata.ts

```diff
@@ -136,7 +145,123 @@ export namespace GdcMetadata {
             whenTriggered: "underThreshold" | "aboveThreshold";
             filterContext?: Uri;
         };
-        meta: IObjectMeta;
+    }
+
+    interface IThemeColorFamily {
+        base: ThemeColor;
+        light?: ThemeColor;
+        dark?: ThemeColor;
+        contrast?: ThemeColor;
+    }
+
+    interface IThemePalette {
+        primary?: IThemeColorFamily;
+        error?: IThemeColorFamily;
+        warning?: IThemeColorFamily;
+        success?: IThemeColorFamily;
+    }
```

Comment:
these two should be exported as well i suspect because they are transitively used by ITheme. I wonder why api-extractor is not complaining (it usually does). hmm